1037-treeview-design-guidance.md

```diff
@@ -0,0 +1,32 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-17
+updated: 2024-10-21
+---
+
+# Treeview Component
+
+[Treeview](https://clarity.design/documentation/treeview) is a hierarchical list of items that have a parent-child relationship where children can be toggled into view by expanding or collapsing their parent item.
+
+## Guidance
+
+- **Must** use a tree view to display and work with large amounts of hierarchical information.
+- **Must** NOT use too many levels unless it is for user-generated information. Instead consolidate the layers if possible.
+- **Must** NOT use for only one level of nested information. Use an `accordion` instead that go one level deep.
```

Comment:
```suggestion
- **Must not** use for only one level of nested information. Use an [accordion](https://clarity.design/documentation/accordion) instead.
```

Reply:
I wonder if this one should be a "should not" instead because our documentation has an example with only one nested level.